WordPress/Jetpack/Resources/AppStoreStrings.po

```diff
@@ -81,17 +81,23 @@ msgctxt "app_store_keywords"
 msgid "social,notes,jetpack,writing,geotagging,media,blog,website,blogging,journal"
 msgstr ""
 
-msgctxt "v24.0-whats-new"
+msgctxt "v24.1-whats-new"
 msgid ""
-"You can now get a WordPress plan with a free domain in the domain dashboard. We also made some improvements to the Site Domains screen so it’s even easier to manage your domains.\n"
+"You’ll now see a notification when you’re working offline. Media uploads to image blocks will also pause when your internet connection cuts out, then resume when you reconnect. Nifty.\n"
 "\n"
-"When you’re editing text blocks in the editor, your device’s keyboard will no longer disappear mid-edit. The editor will also scroll directly to a new block when you add it. Write away, friend.\n"
+"When you select a custom gradient in the block editor, there’s now an indicator to show your chosen color.\n"
 "\n"
-"Miss logging in with your security keys? Not anymore—we’ve re-enabled security keys for two-factor authentication during login. The only one getting into your account is you.\n"
+"Scrolling through your images on the Site Media details screen will make them load ahead of time for a faster experience. Or maybe they were there all along.\n"
 "\n"
-"We updated the colors of the login screen to match both WordPress and Jetpack. After all, teamwork makes the dream work.\n"
+"Image previews are now sized properly when you long-press a file in the media library. We’ve also added support for higher-resolution thumbnails, GIF and video playback, documents, and other file previews. (homer-simpson-woohoo.gif)\n"
```

Comment:
The "sized properly" and the "higher-resolution" thumbnails is referring to the same change.